addons/sourcemod/scripting/shop/player_manager.sp

```diff
@@ -143,8 +143,11 @@ public int PlayerManager_GiveClientItem(Handle plugin, int numParams)
 	}
 	
 	int item_id = GetNativeCell(2);
+	int value = 0;
+	if (numParams == 3)
+		value = GetNativeCell(3);
```

Comment:
Use brackets please `{` & `}` for single line expressions

Reply:
Is it from an aesthetical point of view or for future compatibility? :) 
Brackets were added in the third commit, no problem at all.

Reply:
> Is it from an aesthetical point of view or for future compatibility? :)

1. It's good manners to follow the code style of the project
2. It's common in good projects not to skip brackets.

Thanks !